### ISSUE TITLE
package chopsticks

### ISSUE THREAD

**Issue by andresilva** [andresilva/nix-polkadot#2]

**Comment by dzmitry-lahoda** [comment]
i tried, it is night nightmare

**Comment by dzmitry-lahoda** [comment]
this first to try https://github.com/andresilva/nix-polkadot/issues/4

**Comment by andresilva** [comment]
Yeah, I tried as well and couldn't get it to work due to the usage of workspaces. I also don't have any experience with js project tooling.

**Comment by andresilva** [comment]
Not going to happen.

**Comment by dzmitry-lahoda** [comment]
you reexport zombinet, zombienet is js with workspaces and working fine. basically it is same thing.
